src/main/java/pl/byczazagroda/trackexpensesappbackend/service/WalletService.java

```diff
@@ -0,0 +1,9 @@
+package pl.byczazagroda.trackexpensesappbackend.service;
+
+import pl.byczazagroda.trackexpensesappbackend.model.dto.CreateWalletDto;
+import pl.byczazagroda.trackexpensesappbackend.model.dto.WalletDto;
+
+public interface WalletService {
```

Comment:
If you look up to issue, you can see statement about interface -> "Service interface without method declarations"